### PR TITLE
【prim】instance norm

### PR DESCRIPTION
**Pull request by heyanru01**
### PR types

Others
### PR changes

Others
### Describe

fix norm

### CODE REVIEW

File: python/paddle/incubate/autograd/composite_rules.py [hunk 1]
```diff
@@ -171,6 +171,36 @@ def layernorm_composite(x, scale, bias, epsilon, begin_norm_axis):
     return out, mean_, variance
 
 
+@REGISTER_COMPOSITE('instance_norm')
+def instancenorm_composite(x, scale, bias, epsilon):
+    """
```
Comment (cyber-pioneer) [inline python/paddle/incubate/autograd/composite_rules.py:176]: refer to layer_norm, add amp-relative process.

Comment (heyanru01) [reply python/paddle/incubate/autograd/composite_rules.py:176]: done

**Comment by cyber-pioneer** [comment]
refer to Paddle/python/paddle/fluid/tests/unittests/prim/composite_ops/test_composite_batch_norm.py
add amp_relative test case   and check output Mean, Variance.

refer to Paddle/python/paddle/fluid/tests/unittests/prim/prim/vjp/static/test_comp_batch_norm_grad.py
check weight_grad and bias_grad

**Comment by heyanru01** [comment]
cpu
Max absolute difference: 5.9604645e-07
Max relative difference: 7.694379e-05
gpu
Max absolute difference: 5.9604645e-07
Max relative difference: 0.00281392

**Comment by heyanru01** [comment]
forward：
cpu
Max absolute difference: 5.9604645e-07
Max relative difference: 7.694379e-05
gpu
Max absolute difference: 5.9604645e-07
Max relative difference: 0.00281392

grad：
cpu
Max absolute difference: 3.5667833e-10
Max relative difference: 4.4920656e-10
gpu
Max absolute difference: 9.313226e-10
Max relative difference: 5.820766e-10

**Comment by rainyfly** [comment]
The kernel of this op is not registered for float16

**Pull request closed without merging**